bastet_js/test/Test_JsArray.re

```diff
@@ -21,7 +21,7 @@ module ArbitraryArrayInt:
 };
 
 module TestArray =
-  Test.Array(
+  Test.JsArray(
```

Comment:
Switch this back to using Test.Array (it's cross-platform)

Reply:
If I intentionally break `JsArray` (eg: by flipping the arguments to `Alt.alt`) and switch this to `Test.Array`, all the tests pass.